01-main/README.md

```diff
@@ -154,6 +154,7 @@ cog.out(pretty_list)
 | [<img src="../.github/direct.png" align="top" width="20" />](https://www.ocenaudio.com/) | `ocenaudio` | <i>Easy, fast and powerful audio editor.</i> |
 | [<img src="../.github/debian.png" align="top" width="20" />](https://github.com/jstaf/onedriver) | `onedriver` | <i>A native Linux filesystem for Microsoft Onedrive.</i> |
 | [<img src="../.github/github.png" align="top" width="20" />](https://www.onlyoffice.com/en/desktop.aspx) | `onlyoffice-desktopeditors` | <i>Free desktop office suite for document editing and collaboration.</i> |
+| [<img src="../.github/debian.png" align="top" width="20" />](https://www.speedtest.net/) | `ookla-speedtest` | <i>Official Linux-native Speedtest application backed by Ookla.</i> |
```

Comment:
The README.md is to be updated by maintainers only via `cog`.  See 01-main/CONTRIBUTING.md (Sorry the top-level CONTRIBUTING.md needs correcting)